introduction.md

```diff
@@ -12,33 +12,56 @@ position: 0
+
+The Telerik Report Designers and Viewers are the main components of Telerik Reporting and accompany the users in their journey along the lifecycle of the report. 
 
 ### Report Designers 
 
-The Reporting designers are responsible for the designing process. They are tools which help you create the report you require, connect it to a data source to load the information you need, style its header, footer, and general appearance, make reports collections if necessary, and export its content to PDF, for example. Which designer you will choose depends on the specific functionality you want to use in the process.
+The Report Designers are responsible for the designing process. 
+
+* Report Designers are tools which help you create the report you require, connect it to a data source to load the information you need, style its header, footer, and general appearance, make reports collections if necessary, and export its content to PDF, for example. 
 
-The Telerik Reporting designer tools are: 
+* Which Report Designer you will choose depends on the specific functionality you want to use in the process.
 
-|Telerik Reporting Designer|Specifics
+|Report Designer|Specifics
 |:---|:---
 |Standalone desktop designer|<ul><li>A single `.exe` file representing a Windows tool powered by the Telerik Reporting engine.</li><li>Uses the latest UI standards to facilitate the process of creating reports, locally analyzing report data, exporting and sharing.</li></ul>
 |Web report designer|<ul><li>An HTML5-, JavaScript-, or CSS3 jQuery-based widget allowing you to integrate a report designer into your web applications.</li><li>Styling is based on the Kendo UI Sass Default theme.</li><li>Previews reports in the HTML5 Report Viewer.</li></ul>
 |Designer for Visual Studio|<ul><li>Dedicated to editing CLR or type report definitions (CS or VB files) in the Visual Studio environment.</li><li>Available under the .NET framework only.</li></ul>
 
 ### Report Viewers
 
-The Reporting viewers are UI components which are tailored to a specific technology. Viewers are tools which enable you to deploy and render your report in applications that are built on various technologies such as ASP.NET Core or Angular. Which viewer you will choose to use depends on the targeted technology of the application that will host your report.
+The Report Viewers are UI components which are tailored to a specific technology. 
 
-The Telerik Reporting viewer tools are: 
+* Report Viewers are tools which enable you to deploy and render your report in applications that are built on various technologies such as ASP.NET Core or Angular. 
 
-|Telerik Reporting Viewer|App Type|Essentials
+* Which Report Viewer you will choose to use depends on the targeted technology of the application that will host your report.
+
+|Report Viewer|App Type|Essentials
 |:---|:---|:---
 |HTML5 Report Viewer|Web|An HTML, CSS, or JavaScript client-side widget targeting all web application working with JavaScript. 
```

Comment:
Consider "HTML, CSS, JavaScript +jQuery client-side". Remove 'or'. All these three are part of the viewer. jQuery is also required